projects/core/src/routing/event/routing.events.ts

```diff
@@ -0,0 +1,29 @@
+import { Category, Price } from '../../model/product.model';
+
+export class HomePageVisited {}
+export class CartPageVisited {}
+export class OrderConfirmationPageVisited {}
+export class PageVisited {}
```

Comment:
Can we add a brief API comment for these events?
A one liner would be sufficient, just for the consistency sake.

Reply:
And let's cover `KeywordSearchPageVisited` as well

Reply:
Done.